From 01193d6efbb731dd83eeb9addc14cecb16c431af Mon Sep 17 00:00:00 2001
From: Lukas Tribus <luky-37@hotmail.com>
Date: Sat, 12 Apr 2014 18:41:02 +0200
Subject: [PATCH 124/139] BUG/MINOR: build: add missing objects in osx and bsd
 Makefiles

Commits 98634f0c7b8, ed66c297c26 and d5f624dde78 updated the main
makefile with new objects src/hash.o, src/pattern.o and src/map.o,
but missed to update Makefile.bsd and Makefile.osx.

This will fix some of the build failures reported by Jorge Severino
under OpenBSD.
---
 Makefile.bsd | 3 ++-
 Makefile.osx | 3 ++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/Makefile.bsd b/Makefile.bsd
index ac8bfd6..0e56077 100644
--- a/Makefile.bsd
+++ b/Makefile.bsd
@@ -116,7 +116,8 @@ OBJS = src/haproxy.o src/sessionhash.o src/base64.o src/protocol.o \
        src/lb_chash.o src/lb_fwlc.o src/lb_fwrr.o src/lb_map.o src/lb_fas.o \
        src/ev_poll.o src/ev_kqueue.o src/connection.o \
        src/arg.o src/acl.o src/memory.o src/freq_ctr.o src/payload.o \
-       src/auth.o src/stick_table.o src/sample.o src/compression.o
+       src/auth.o src/stick_table.o src/sample.o src/compression.o \
+       src/hash.o src/pattern.o src/map.o
 
 EBTREE_OBJS = $(EBTREE_DIR)/ebtree.o \
               $(EBTREE_DIR)/eb32tree.o $(EBTREE_DIR)/eb64tree.o \
diff --git a/Makefile.osx b/Makefile.osx
index 0348b86..25d28a1 100644
--- a/Makefile.osx
+++ b/Makefile.osx
@@ -113,7 +113,8 @@ OBJS = src/haproxy.o src/sessionhash.o src/base64.o src/protocol.o \
        src/lb_chash.o src/lb_fwlc.o src/lb_fwrr.o src/lb_map.o src/lb_fas.o \
        src/ev_poll.o src/connection.o src/payload.o \
        src/arg.o src/acl.o src/memory.o src/freq_ctr.o \
-       src/auth.o src/stick_table.o src/sample.o src/compression.o
+       src/auth.o src/stick_table.o src/sample.o src/compression.o \
+       src/hash.o src/pattern.o src/map.o
 
 EBTREE_OBJS = $(EBTREE_DIR)/ebtree.o \
               $(EBTREE_DIR)/eb32tree.o $(EBTREE_DIR)/eb64tree.o \
-- 
1.8.3.2

